demo-kotlin/app/src/main/java/com/example/placesdemo/CurrentPlaceActivity.kt

```diff
@@ -82,31 +108,27 @@ class CurrentPlaceActivity : AppCompatActivity() {
     }
 
     /**
-     * Fetches a list of [PlaceLikelihood] instances that represent the Places the user is
-     * most
-     * likely to be at currently.
+     * Check whether permissions have been granted or not, and ultimately proceeds to either
+     * request them or runs {@link #findCurrentPlaceWithPermissions() findCurrentPlaceWithPermissions}
      */
     private fun findCurrentPlace() {
-        if (ContextCompat.checkSelfPermission(this, permission.ACCESS_WIFI_STATE)
-            != PackageManager.PERMISSION_GRANTED
-            || ContextCompat.checkSelfPermission(this, permission.ACCESS_FINE_LOCATION)
-            != PackageManager.PERMISSION_GRANTED) {
-            Toast.makeText(
-                this,
-                "Both ACCESS_WIFI_STATE & ACCESS_FINE_LOCATION permissions are required",
-                Toast.LENGTH_SHORT)
-                .show()
-        }
+        when (PackageManager.PERMISSION_GRANTED) {
+            ContextCompat.checkSelfPermission(
+                applicationContext,
+                permission.ACCESS_COARSE_LOCATION
```

Comment:
We need to check if either ACCESS_COARSE_LOCATION or ACCESS_FINE_LOCATION is granted. Is there a check like "hasOnePermissionGranted" used in [L149](https://github.com/googlemaps/android-places-demos/pull/348/files#diff-ffc8f0109e203da82714446ad27d274277c3e2c5893e110e0f6a6fe72bd9ecf2R149)?

Reply:
Done.